### DISCUSSION TITLE
Better BibTex automatic export question

### DISCUSSION THREAD

**Discussion by rkrug** [retorquere/zotero-better-bibtex#2586] [General]
Hi

think Better BibTex is **the** add-on for Zotero - an absolute must! Thanks.

I would like to confirm two points:
1. I suspect, that Better BibTex does not change / add anything in any items in a Group in default settings (e.g. not pinning the BibTex key)? 
2. Automatic Export to BibTex works also for groups? (I remember, It didn't)?

and one question:

Is there any way of identifying, when an automatic export is in progress?

I am asking, because I would like to make automatic backups of several critical groups, and will avoid making backups during an ongoing automatic export.

Thanks,

Rainer

**Comment by retorquere** [comment]
1. Automatic citekey pinning is off by default. This might change if zotero adds its own citekey field in the not too far future.

2. There's not really a difference between the library and groups, so this has always worked.

**Reply by rkrug** [reply]
Thanks. Sounds perfect.
